var/spack/repos/builtin/packages/poppler/package.py

```diff
@@ -10,21 +10,22 @@ class Poppler(CMakePackage):
     """Poppler is a PDF rendering library based on the xpdf-3.0 code base."""
 
     homepage = "https://poppler.freedesktop.org"
-    url      = "https://poppler.freedesktop.org/poppler-0.72.0.tar.xz"
+    url      = "https://poppler.freedesktop.org/poppler-0.77.0.tar.xz"
     list_url = "https://poppler.freedesktop.org/releases.html"
 
+    version('0.77.0', sha256='7267eb4cbccd64a58244b8211603c1c1b6bf32c7f6a4ced2642865346102f36b')
     version('0.72.0', sha256='c1747eb8f26e9e753c4001ed951db2896edc1021b6d0f547a0bd2a27c30ada51')
     version('0.65.0', 'b9a0af02e43deb26265f130343e90d78')
     version('0.64.0', 'f7f687ebb60004f8ad61994575018044')
 
     variant('cms',      default=False, description='Use color management system')
+    variant('cpp',      default=False, description='Compile poppler cpp wrapper')
     variant('glib',     default=False, description='Compile poppler glib wrapper')
     variant('gobject',  default=False, description='Generate GObject introspection')
     variant('libcurl',  default=False, description='Build libcurl based HTTP support')
     variant('openjpeg', default=False, description='Use libopenjpeg for JPX streams')
     variant('qt5',      default=False, description='Compile poppler qt5 wrapper')
     variant('zlib',     default=False, description='Build with zlib')
-    variant('cairo',    default=False, description='Search for Cairo package')
```

Comment:
From what I can tell, if you want to build `poppler-glib`, you need both `glib` and `cairo`. I don't see an option to only use `cairo` on its own.